feelslikemonday/app/src/main/java/com/example/feelslikemonday/ui/following/FollowingPageAdapter.java

```diff
@@ -52,6 +67,25 @@ public View getView(int position, @Nullable View convertView, @NonNull ViewGroup
         tvTime.setText(timePosted);
         tvEmotion.setText(mood);
 
+        if (moodname.equals("Anger")){
```

Comment:
Is it possible to make the colors slightly lighter and slightly less saturated, so that is matches the light green look that you gave the app? Your call